beanstalkd-console/config.php

```diff
@@ -0,0 +1,23 @@
+<?php
+$GLOBALS['config'] = [
+    /**
+     * List of servers available for all users
+     */
+    'servers' => ['beanstalkd:11300' => 'beanstalkd:11300'],
+    /**
+     * Saved samples jobs are kept in this file, must be writable
+     */
+    'storage' => dirname(__FILE__) . DIRECTORY_SEPARATOR . 'storage.json',
+    /**
+     * Optional Basic Authentication
+     */
+    'auth'    => [
+        'enabled'  => false,
+        'username' => 'admin',
+        'password' => 'password',
+    ],
+    /**
+     * Version number
+     */
+    'version' => '1.7.10',
```

Comment:
I think it should not be hardcoded that way, because currently we are pulling the latest version from `master`
`https://github.com/ptrofimov/beanstalk_console/archive/master.tar.gz`